static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -142,3 +184,22 @@ function SmallWidth({feedbackItem}: {feedbackItem: FeedbackIssue}) {
     />
   );
 }
+
+const openDeleteModal = onDelete =>
+  openModal(({Body, Footer, closeModal}: ModalRenderProps) => (
+    <Fragment>
+      <Body>
+        {t('Deleting this feedback is permanent. Are you sure you wish to continue?')}
+      </Body>
+      <Footer>
+        <Button onClick={closeModal}>{t('Cancel')}</Button>
+        <Button
+          style={{marginLeft: space(1)}}
```

Comment:
Let's make this a styled component instead of using inline styles

Reply:
could have a `display: flex; gap: space(1);` wrapper instead of using margin if we're touching it. i <3 flex.